spikeinterface/sortingcomponents/peak_detection.py

```diff
@@ -173,7 +283,7 @@ def _detect_peaks_chunk(segment_index, start_frame, end_frame, worker_ctx):
     # load trace in memory
     recording_segment = recording._recording_segments[segment_index]
     traces, left_margin, right_margin = get_chunk_with_margin(recording_segment, start_frame, end_frame, 
-                                                              None, margin, add_zeros=True)
+                                                              slice(None), margin, add_zeros=True)
```

Comment:
please no. This is outside the PR.
Furthermore, you should make the fix inside the function and not outside.

Reply:
deal, removed